autoload/dirvish.vim

```diff
@@ -387,7 +387,7 @@ function! dirvish#open(...) range abort
     call s:msg_error("'autochdir' is not supported")
     return
   endif
-  if !&hidden && &modified
+  if !&hidden && &modified && len(win_findbuf(winbufnr(0))) == 1
```

Comment:
Ah.  It looks like `winbufnr` has been there since at least 7.0, but `win_findbuf` was only added in 7.4.1558.  I'll add an `exists()` check.